samples/csharp/getting-started/MatrixFactorization_MovieRecommendation/MovieRecommendation/Program.cs

```diff
@@ -54,11 +63,11 @@ static void Main(string[] args)
 
             //STEP 6: Evaluate the model performance 
             Console.WriteLine("=============== Evaluating the model ===============");
-            IDataView testDataView = mlcontext.Data.ReadFromTextFile<MovieRating>(TestDataLocation, hasHeader: true); 
+            IDataView testDataView = mlcontext.Data.ReadFromTextFile<MovieRating>(TestDataLocation, hasHeader: true, separatorChar: ','); 
```

Comment:
I thought I pushed this changed for issue in this commit https://github.com/dotnet/machinelearning-samples/commit/501b49f7e773a448685f266640a241befd0ff88d Did not understand why it was not in master.